### PR TITLE
Feature/add license

### PR DESCRIPTION
**Pull request by allardvanhelbergen**
- Add functionality for GraphQL pages.
- Add LICENSE.md.
- Add Terms and policies page from markdown.
- Refactor social buttons, mailchimp form and footer to be components.

### CODE REVIEW

File: src/content/terms-and-policies.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
+---
```
Comment (paulwyn) [inline src/content/terms-and-policies.md:1]: Is this a standard terms and policies document that you copied over from somewhere? And did you read all of it?

File: src/pages/index.js [hunk 2]
```diff
@@ -1,11 +1,7 @@
 import React from 'react';
```
Comment (paulwyn) [inline src/pages/index.js:1]: Is this deployed somewhere that I can take a look?

**Review by paulwyn — APPROVED** [review]

File: src/components/SocialButtons.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+fontawesome.library.add(faEnvelope, faTwitter);
+
+
+const SocialButtons = () => (
+  <div className="social-icons">
+    <ul className="list-unstyled text-center mb-0">
+      <li className="list-unstyled-item">
+        <a href="#">
+          {/* <FontAwesomeIcon icon="coffee" /> */}
```
Comment (paulwyn) [inline src/components/SocialButtons.js]: Comments? Remove pls

File: src/content/terms-and-policies.md (hunk 1 above)
Comment (allardvanhelbergen) [reply src/content/terms-and-policies.md:1]: I wrote it from scratch!

File: src/pages/index.js (hunk 2 above)
Comment (allardvanhelbergen) [reply src/pages/index.js:1]: I deployed it to your local machine when you were in the shower.

File: src/components/SocialButtons.js (hunk 3 above)
Comment (allardvanhelbergen) [reply src/components/SocialButtons.js]: ok... fine.

**Pull request merged**